### PR TITLE
Lock down last stable ethereum test suite for aevm tests

### PR DESCRIPTION
**Pull request by gorillainduction**
https://www.pivotaltracker.com/n/projects/2124891/stories/157550448

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -193,6 +194,11 @@ aevm-test: aevm-test-deps
 	@./rebar3 eunit --application=aevm
 
 aevm-test-deps: $(AEVM_EXTERNAL_TEST_DIR)/ethereum_tests
+aevm-test-deps:
+	$(eval VER=$(shell git -C "$(AEVM_EXTERNAL_TEST_DIR)/ethereum_tests" log -1 --pretty=format:"%H"))
+	@if [ "$(VER)" != "$(AEVM_EXTERNAL_TEST_VERSION)" ]; then \
```
Comment (dincho) [inline Makefile]: I'm not sure why we'd need this explicit check, I guess git does that internal and much faster 🤔

Comment (gorillainduction) [reply Makefile]: You are right. I just wanted to be a bit more silent if everything was ok.

File: Makefile [hunk 2]
```diff
@@ -193,6 +194,11 @@ aevm-test: aevm-test-deps
 	@./rebar3 eunit --application=aevm
 
 aevm-test-deps: $(AEVM_EXTERNAL_TEST_DIR)/ethereum_tests
+aevm-test-deps:
+	$(eval VER=$(shell git -C "$(AEVM_EXTERNAL_TEST_DIR)/ethereum_tests" log -1 --pretty=format:"%H"))
```
Comment (lucafavatella) [inline Makefile]: `git rev-parse HEAD`?

Comment (gorillainduction) [reply Makefile]: As this is a stopgap, I'll not update. But indeed. That might have been more clear.

**Pull request merged**